include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
Actually, thinking it over, I suggest to remove the #ifdef completely. Reason is that when using CppUTest as a library, then it is not unlikely that the header is included without the CPPUTEST_USE_LONG_LONG define. So they'll get an error, *even though* they might have compiler CppUTest with long long support. So, this #ifdef might force the user of CppUTest to #define USE_LONG_LONG